openreview/venue/group.py

```diff
@@ -291,6 +291,9 @@ def create_venue_group(self):
         if self.venue.source_submissions_query_mapping:
             content['source_submissions_query_mapping'] = { 'value': self.venue.source_submissions_query_mapping }    
 
+        if self.venue.reviewer_assignment_quota:
```

Comment:
If this property is going to be used in the ARR venue only then we don't need to save it in the venue code. You should do it for the ARR venues only.

If this property is going to be used by other venues too then we should implement it in the pre processes of the venue class too.